index.d.ts

```diff
@@ -309,7 +308,9 @@ const cli = meow(`
 foo(cli.input[0], cli.flags);
 ```
 */
+// eslint-disable-next-line no-redeclare
 declare function meow<Flags extends meow.AnyFlags>(helpMessage: string, options?: meow.Options<Flags>): meow.Result<Flags>;
+// eslint-disable-next-line no-redeclare
 declare function meow<Flags extends meow.AnyFlags>(options?: meow.Options<Flags>): meow.Result<Flags>;
```

Comment:
Just use `export default` on both of these. No need for declare.

Reply:
Do you mean use `export default` twice like this?

```typescript
export default function meow<Flags extends AnyFlags>(helpMessage: string, options?: Options<Flags>): Result<Flags>;
export default function meow<Flags extends AnyFlags>(options?: Options<Flags>): Result<Flags>;
```